Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -255,8 +255,8 @@ struct CombinatorialKalmanFilterResult {
 
   Result<void> result{Result<void>::success()};
 
-  // TODO place into options and make them accessible?
-  AbortList<PathLimitReached, EndOfWorldReached> abortList;
```

Comment:
So this is now explicit and separate rather than having the `AbortList` handle it?

Reply:
I found the `AbortList` not clear because we check different things in different places. this is why I took it apart